src/main/java/com/github/minecraft_ta/totaldebug/command/DecompileCommand.java

```diff
@@ -0,0 +1,82 @@
+package com.github.minecraft_ta.totaldebug.command;
+
+import com.github.minecraft_ta.totaldebug.TotalDebug;
+import net.minecraft.block.Block;
+import net.minecraft.command.*;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityList;
+import net.minecraft.item.Item;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.registry.EntityEntry;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+public class DecompileCommand extends CommandBase {
+
+    @Override
+    public String getName() {
+        return "decompile";
+    }
+
+    @Override
+    public String getUsage(ICommandSender sender) {
+        return "commands.total_debug.decompile.usage";
+    }
+
+    @Override
+    public List<String> getTabCompletions(MinecraftServer server, ICommandSender sender, String[] args, @Nullable BlockPos targetPos) {
+        if (args.length == 1) {
+            return getListOfStringsMatchingLastWord(args, "item", "block", "entity", "path");
+        } else if (args.length == 2) {
+            switch (args[0]) {
```

Comment:
Please add autocomplete for paths @tth05

Reply:
done